### PR TITLE
Changes to resolve the symbol conflict because of gf_mul

### PR DESCRIPTION
**Pull request by Pankul94**
CLA: trivial

This PR is a result of discussion on resolving `gf_mul` symbol conflict when using intel/ISA-L and Openssl, https://github.com/openssl/openssl/issues/21302.

### CODE REVIEW

**Comment by t8m** [comment]
> Since these changes are just renaming internal functions, can we consider this trivial and not need the CLA check.

Please see https://www.openssl.org/policies/cla.html how to mark the commit with CLA: trivial - it needs to be written in the commit message body on a separate line.

File: crypto/ec/curve448/arch_32/f_impl32.c [hunk 1]
```diff
@@ -100,6 +100,6 @@ void gf_mulw_unsigned(gf_s * RESTRICT cs, const gf as, uint32_t b)
 
 void gf_sqr(gf_s * RESTRICT cs, const gf as)
```
Comment (t8m) [inline crypto/ec/curve448/arch_32/f_impl32.c:101]: Could you please rename also this one?

Comment (Pankul94) [reply crypto/ec/curve448/arch_32/f_impl32.c]: I will follow the same standard, `ossl_gf_sqr`

**Comment by t8m** [comment]
Please add the `CLA: trivial` annotation to the commit message if you want to submit it without CLA.

As this is pure rename of 3 symbols I think it is acceptable as CLA: trivial.

**Comment by Pankul94** [comment]
Added CLA: Trivial to the git commit message

**Review by kroeckx — APPROVED** [review]

**Comment by t8m** [comment]
@kroeckx please confirm you're OK with CLA: trivial for this.

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
Merged.  Thanks for the contribution.

**Comment by baibaichen** [comment]
Would we cherry pick it to 3.0 and 3.1 branch?

**Comment by t8m** [comment]
Please submit a PR against 3.1 branch.

**Pull request closed without merging**